internal/ui/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@unkey/ui",
+  "description": "UI components for Unkey",
+  "version": "0.1.1",
+  "sideEffects": false,
+  "main": "./src/index.ts",
+  "types": "./src/index.ts",
+  "scripts": {
+    "lint": "eslint src/",
+    "check-types": "tsc --noEmit"
+  },
+  "peerDependencies": {
+    "next": "14.2.10",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@unkey/tsconfig": "workspace:^",
+    "@types/react": "^18.2.47",
+    "@types/react-dom": "^18.2.14",
+    "autoprefixer": "^10.4.19",
+    "next": "14.2.10",
+    "postcss": "^8.4.38",
+    "react": "^18.2.0",
+    "tailwindcss": "^3.4.3",
+    "typescript": "^5.5.3"
+  },
+  "dependencies": {},
+  "author": "Nazar Poshtarenko <renkonazbka@gmail.com>",
```

Comment:
btw, not sure about this line. let me know if that’s okay or if you prefer it limited to core contributors.

Reply:
and since it's an internal package, one may find this info excessive. I'm ok with refining this file to keep it clean and consistent w/ other packages